packages/kbn-securitysolution-es-utils/src/get_index_exists/index.ts

```diff
@@ -8,6 +8,7 @@
 
 import type { ElasticsearchClient } from '../elasticsearch_client';
 
+// TODO: Being fooled by alias, so can return true if concrete index doesn't exist
```

Comment:
seems worthy of a new issue?  We probably need a similar kind of check in event log?

Reply:
Yeah, this is outdated and just needs to be removed. There was a work-around necessary back when I first implemented that needed to be done for things to play nice with the RuleRegistry, but that has since been resolved, so we should be good here 👍